cpp/modmesh/toggle/pymod/wrap_profile.cpp

```diff
@@ -68,104 +68,6 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapWrapperProfilerStatus
 
 }; /* end class WrapWrapperTimerStatus */
 
-class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapStopWatch
```

Comment:
Please keep `StopWatch` which is a convenient help for quick timing and add corresponding unit tests.